source/index.html

```diff
@@ -0,0 +1,5 @@
+<!DOCTYPE html>
+<head>
+  <title>Impasta Rosta</title>
+</head>
+<body></body>
```

Comment:
Add `html` tags and add the following metadata to `head`

```suggestion
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Impasta Rosta</title>
  </head>

  <body></body>
</html>
```